INSTALL.md

```diff
@@ -46,6 +52,12 @@ sudo dnf install -y \
     wget git
 ```
 
+Including for libsubid support:
+
+```sh
+sudo dnf --enablerepo=powertools install shadow-utils-subid-devel
```

Comment:
Unfortunately as you pointed out, it's in the crb repo on el9, not powertools.  So powertools doesn't work for both.  So I'm sorry for the churn, but I guess it is better to use `--enablerepo=devel` after all because it will work on both.  For Fedora the devel repo doesn't exist, and also there aren't powertools or crb repos.  So the instructions here I guess should say to use `--enablerepo=devel` for el8 and el9 but not for fedora.

Reply:
No worries! should be all fixed now.